### PR TITLE
Updated TFTS docs

### PR DESCRIPTION
**Pull request by kahaaga**

### CODE REVIEW

**Comment by kahaaga** [comment]
@datseries There are some design choices to be made here. I detailed them on the slack channel.

**Comment by Datseris** [comment]
Don't worry about the failing docs, I'll take care of them. Merge this when it is alright to you and I'll put some work into the docs PR

**Comment by Datseris** [comment]
The new docs are finally up and operational: https://juliadynamics.github.io/TimeseriesSurrogates.jl/dev/

The documentation part of this PR should be adjusted to the new docs.

File: src/methods/truncated_fourier.jl [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+"""
+struct TFTS <: Surrogate
+    fϵ # ratio of high-frequency domain to the whole domain
+
+    function TFTS(fϵ::Real)
+        fϵ >= 0 || error("`fϵ` must be >= 0")
+        new(fϵ)
+    end
+end
+TFTS() = error("The truncation factor `fϵ` must be supplied")
```
Comment (Datseris) [inline src/methods/truncated_fourier.jl:36]: I don't think this line of code should be implemented (for source code sanity). Adding an error message for possible absence of arguments doesn't make sense: it is mandatory to read the documentation string to use any function, and the docstring tells you the arguments.

**Pull request merged**